Machine Learning/Regression Loss Functions/regression-loss-functions.ipynb

```diff
@@ -248,17 +316,19 @@
     "\n",
     "The distribution of the values has a \"bump\" for smaller values and then a long tail with a small number of very large values. \n",
     "\n",
-    "Naturally, when we want to predict the target values, we want to allow for an error depending on the magnitude of the target. It’s ok to miss a population of a metropolitan area by 10 000, but for a small town it is unforgivable. In this case MSLE can be more suitable for the regression job than MSE. As the name suggests, MSLE minimizes the quadratic difference between the logarithms of the actual value and the prediction\n",
+    "Naturally, when you want to predict the target values, you want to allow for an error depending on the magnitude of the target. It’s ok to miss a population of a metropolitan area by 10 000, but for a small town it is unforgivable. In this case MSLE can be more suitable for the regression job than MSE. As the name suggests, MSLE minimizes the quadratic difference between the logarithms of the actual value and the prediction\n",
     "$$L(a_i, p_i) = (\\log(a_i + t) - \\log(p_i + t))^2$$\n",
     "\n",
-    "Qualitatively speaking, MSLE tends to lead to lower error for small targets and for higher error on large targets than MSE. This trade-off can be controlled by adjusting an offset parameter $t$ which is set to 1.0 by default. By increasing this parameter we influence the “transition point” at which we go from minimizing quadratic error to minimizing quadratic log error. \n",
+    "For example, $\\log\\left((1 + 1100) / (1 + 1000)\\right)^2$ is about the same as $\\log((1 + 11000) / (1 + 10000))^2$ so missing a value by 10\\% is penalised equivalently whether the actual error is 100 or 1000. At the same time, with MSE the latter case costs 100x more.\n",
```

Comment:
Maybe to carry on the earlier example you could use `log((1 + 110000) / (1 + 100000)` is about the same as `log((1 + 11000) / (1 + 10000))^2`, i.e. this relates directly back to the population example.